src/fields-to-params.js

```diff
@@ -1,16 +1,24 @@
 export default function fieldsToParams(fieldsObject) {
-  return Object.keys(fieldsObject)
-    .filter(fieldName =>
-      Object.prototype.hasOwnProperty.call(FIELDS_TO_PARAMS_MAP, fieldName)
-    )
-    .filter(fieldName => fieldsObject[fieldName])
-    .reduce(
-      (obj, fieldName) => ({
-        ...obj,
-        [FIELDS_TO_PARAMS_MAP[fieldName]]: fieldsObject[fieldName]
-      }),
-      {}
-    )
+  const params = {}
+  for (const fieldName of Object.keys(fieldsObject)) {
```

Comment:
> I couldn't figure out how to make it work as a map/reduce, so I… used a for-loop. 😆

Sometimes a for-loop is indeed better than a bag of functions 😄

You could have used `for (const fieldName in fieldsObject)` to avoid calling `Object.keys`, but that's a small nitpick, I'll take care of it.